initfiles/examples/embed/python_cat.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 from __future__ import print_function
```

Comment:
You don't need this any more if you are using python3

Reply:
But on my CentOS 8 get error "ambiguous python shebang"